app/ide-desktop/lib/dashboard/src/layouts/dashboard/DataLinkInput.tsx

```diff
@@ -148,7 +156,9 @@ export default function DataLinkInput(props: DataLinkInputProps) {
               value={typeof value === 'string' ? value : ''}
               size={1}
               className={`rounded-full w-40 px-2 bg-transparent border leading-170 h-6 py-px disabled:opacity-50 read-only:opacity-75 read-only:cursor-not-allowed ${
-                jsonSchema.isMatch(DEFS, schema, value) ? 'border-black/10' : 'border-red-700/60'
+                error.assert<ajv.ValidateFunction>(() => AJV.getSchema(path))(value)
```

Comment:
I think I'd refactor this `error.assert<...>(() => AJV.getSchema(path))` pattern into some helper function, as it is used all over the place.

e.g. `function getKnownSchema`